web/src/components/Contact.tsx

```diff
@@ -22,23 +23,10 @@ const Root = styled('div')({
   }
 })
 
-interface Props {
-  productName: string
-}
-
-const Contact = (props: Props) => {
-  const productName = props.productName
-
+const Contact = () => {
```

Comment:
Can we get rid of this `Contact` component entirely and just import the `FeedbackButton` into the `GeneralHeader` component? Or if you want to keep the `Contact` component there is no need for the `Root` styled div (can be deleted along with the classes.